### PR TITLE
chore: create docker compose dev env

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3577212&stafilter=NotComplete&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      db:
+        condition: service_healthy
+  redis:
+    image: redis:latest
+    restart: always
+    command: redis-server --save 20 1 --loglevel warning --requirepass ${REDIS_PASSWORD}
+    ports:
+      - ${REDIS_PORT}
+    volumes:
+      - /tmp/redis:/root/redis
```
Comment (smarcet) [inline docker-compose.yml]: make /tmp/summit_reports_api/redis

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      MYSQL_PASSWORD: ${DB_OPENSTACK_PASSWORD}
+      MYSQL_ROOT_PASSWORD: ${DB_OPENSTACK_PASSWORD}
+      SERVICE_TAGS: dev
+      SERVICE_NAME: mysql
+    healthcheck:
+      test: [ "CMD", "mysqladmin" ,"ping", "-h", "localhost" ]
+      timeout: 20s
+      retries: 10
+    volumes:
+      - /tmp/mysql/${DB_OPENSTACK_NAME}:/var/lib/mysql
```
Comment (smarcet) [inline docker-compose.yml]: do /tmp/summit_reports_api/mysql

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review the comments

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: summit-report-api
+    command: python manage.py runserver 0.0.0.0:8000
```
Comment (smarcet) [inline docker-compose.yml]: change the port to something diff than api and idp

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: docker-compose.yml (hunk 3 above)
Comment (romanetar) [reply docker-compose.yml]: done. New port is 8003.

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**